### PR TITLE
[MINT-2720] Add basic data exchange seed data

### PR DESCRIPTION
**Pull request by ClayBenson94**
# MINT-2720

## Description

- Adds data exchange seed data
  - Currently doesn't add many variations of data (different statuses, different answers), but this is just aiming to have _some_ seed data in place.
- Updates postman collection for a method to fetch data exchange, rather than just update.

## How to test this change

1.  Run `scripts/dev up`, followed by `scripts/dev db:seed`
2. Open the UI and find the model with data exchange info, copying it's `id`
3. Run the new postman query to fetch data exchange with that ID, ensuring all data is correctly populated.

## PR Author Checklist

- [ ] I have provided a detailed description of the changes in this PR.
- [ ] I have provided clear instructions on how to test the changes in this PR.
- [ ] I have updated tests or written new tests as appropriate in this PR.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.

### CODE REVIEW

File: cmd/dbseed/resolver_wrappers.go
Comment (OddTomBrooks) [inline cmd/dbseed/resolver_wrappers.go:103]: Might be worth waiting until notification is merged since the call spec is differentiated

**Review by OddTomBrooks — APPROVED** [review]
LGTM, I would suggest holding off for the notification merge since it changes the call spec

File: cmd/dbseed/resolver_wrappers.go
Comment (ClayBenson94) [reply cmd/dbseed/resolver_wrappers.go:103]: Yeah, I think I'll wait for the notification PR, then update it here rather than the other way around!

Comment (ClayBenson94) [reply cmd/dbseed/resolver_wrappers.go]: @OddTomBrooks 5e6e1959 just was pushed to resolve this. 

Currently set to hard-code email args to `nil` or `models.AddressBook{}` since we never really send emails with seed data anyways

**Pull request merged**
